packages/goflow2/data_stream/sflow/manifest.yml

```
@@ -0,0 +1,61 @@
+title: "Goflow2 sFlow"
+type: logs
+streams:
+  - input: logfile
```

The `logfile` input is an alias for `log`, which has been deprecated. Use `filestream` instead. It should be a drop-in replacement for `logfile`, but feel free to reference something like [f5_bigip](https://github.com/elastic/integrations/blob/main/packages/f5_bigip/data_stream/log/agent/stream/log.yml.hbs) as an example (even though the file is log.yml.hbs, it is using filestream).

```suggestion
  - input: filestream
```

---

fixed in https://github.com/elastic/integrations/pull/10561/commits/c8c8a0271f0d15b2d0d046037fd492e8ba05aef7